### ISSUE TITLE
No separator between views and time (after #3906)

### ISSUE THREAD

**Issue by luckkmaxx** [libre-tube/LibreTube#3910]
### Steps to reproduce

#3906 Missed to keep separation or at least a space between views and time info

### Expected behavior

like before the fix

### Actual behavior

no spaces
![Screenshot_20230603-132241_LibreTube Debug](https://github.com/libre-tube/LibreTube/assets/14180594/9c2b583f-c5a2-4cb9-8381-276916cff09b)

### LibreTube version

0.5.0 debug

### Android version

13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by luckkmaxx** [comment]
I was thinking I could abbreviate the "300 views" wich in spanish is set to "300 reproducciones". So a shorter expression will give space to the time info: "300 reprod."
I have no more ideas, and this cant fix it in other languages if "views" is a larger word.

**Comment by Bnyro** [comment]
Whoopsie, each commit seems to introduce a regression today :wink:
